src/app/public/modules/type-definitions/anchor-link-service-config.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Specifies the configuration options to create anchor links.
+ */
+export interface SkyAnchorLinkConfig {
```

Comment:
For consistency, `SkyDocsAnchorLinkConfig`.

Reply:
Fixed.